package.json

```diff
@@ -116,6 +117,7 @@
     "cover:unit": "node --max-old-space-size=4096 --harmony --trace-deprecation ./node_modules/istanbul/lib/cli.js cover --report none node_modules/mocha/bin/_mocha -- test/*.unittest.js",
     "cover:report": "istanbul report",
     "cover:report-min": "istanbul report --report lcovonly",
-    "publish-patch": "npm run lint && npm run beautify-lint && mocha && npm version patch && git push && git push --tags && npm publish"
+    "publish-patch": "npm run lint && npm run beautify-lint && mocha && npm version patch && git push && git push --tags && npm publish",
+    "webpack-defaults": "webpack-defaults"
```

Comment:
Why is this needed ?  That's the boilerplate for repos we use at `webpack-contrib` which doesn't apply here :)

Reply:
@michael-ciniawsky Sorry! Didn't notice this change at first. I removed it in the latest commit, this is no longer in the branch.

When I was testing with the `schema-utils` installed from the git tag version, I had to build it manually inside of `node_modules` and somehow build step added this line to `package.json`, still don't really understand why this happened 😅